proposals/content-addressed-pkg-mgmt.md

```diff
@@ -0,0 +1,164 @@
+- A published draft of a v0.1 of the content addressed package management protocol specification
+- A functional implementation of the specification with support for multiple package managers
+- Documented discussion of protocol and process with a selection of package manager maintainers, publishers and consumers
+
+####  What does success look like?
+_Success means impact. How will we know we did the right thing?_
+
+<!--
+Provide success criteria. These might include particular metrics, desired changes in the types of bug reports being filed, desired changes in qualitative user feedback (measured via surveys, etc), etc.
+-->
+- Increasing number of users sharing packages on the dht
+- Projects and package managers adopting and building on the protocol
+- Increased discussion around content addressing in software development and tooling
+
+#### Counterpoints &amp; pre-mortem
+_Why might this project be lower impact than expected? How could this project fail to complete, or fail to be successful?_
+
+- Content discovery and publishing on the IPFS DHT of millions of packages may affect growth once a certain level of adoption is reached
+- If the user experience is poor or switching costs are too high, reaching critical levels of adoption will be difficult
+- There may be incentives for some package management infrastructure organizations to retain centralized control and choose to avoid adoption
+
+#### Alternatives
+_How might this project’s intent be realized in other ways (other than this project proposal)? What other potential solutions can address the same need?_
+
+- [Valist](https://github.com/valist-io/valist) is an alternative project, although it appears to be closely tied to Ethereum.
+
+#### Dependencies/prerequisites
+<!--List any other projects that are dependencies/prerequisites for this project that is being pitched.-->
+
+- Whilst not directly dependent on, https://github.com/protocol/beyond-bitswap/pull/29 can help ease onboarding 
+
+#### Future opportunities
+<!--What future projects/opportunities could this project enable?-->
+A standardized package metadata format can enable the development of:
+- a content addressable, public transparency log for all package managers, similar to the ]certificate transparency project](https://certificate.transparency.dev/)
+- package maintainers could publish signed nfts of each release of their packages, enabling alternative sources of funding for open source
+
+## Required resources
+
+#### Effort estimate
+<!--T-shirt size rating of the size of the project. If the project might require external collaborators/teams, please note in the roles/skills section below).
+For a team of 3-5 people with the appropriate skills:
+- Small, 1-2 weeks
+- Medium, 3-5 weeks
+- Large, 6-10 weeks
+- XLarge, >10 weeks
+Describe any choices and uncertainty in this scope estimate. (E.g. Uncertainty in the scope until design work is complete, low uncertainty in execution thereafter.)
+-->
+
+- Large, 6-10 weeks
```

Comment:
Do you have a clear enough picture of the steps involved in this to be able to give a rough indication of the relative sizes of what's listed in `Brief plan of attack`?
It also might be worth considering if there are 2 or 3 discrete sub-projects this could be split up into such that just doing one would provide value in itself while also serving as a way to justify further investigation.